app/models/manageiq/providers/openstack/cloud_manager/vm/associate_ip.rb

```diff
@@ -0,0 +1,47 @@
+module ManageIQ::Providers::Openstack::CloudManager::Vm::AssociateIp
+  extend ActiveSupport::Concern
+
+  included do
+    supports :associate_floating_ip do
+      if cloud_tenant.nil? || Rbac.filtered(cloud_tenant.floating_ips).empty?
```

Comment:
@kbrock when you say "add [floating ips] as a class in rbac" are you referring to the list in `lib/rbac/filterer.rb`? There doesn't seem to be much documentation on this topic floating around.